src/Qgis.xsd

```diff
@@ -29,7 +29,6 @@
 			<xs:sequence>
 				<xs:element name="MapLayer" type="MapLayer" />
 			</xs:sequence>
-			<xs:attribute name="LayerCount" type="xs:integer" />
```

Comment:
Is this file a common ressource? Could this change break older versions?

It apparently does not break the loading of old projects for this version of QGIS. And the URL http://mrcc.com/Qgis.xsd referenced in the file itself produces 404 anyway.